<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTableWidget" name="lizenzTable">
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Dies ist ein kleines Tool zum Verwalten von Lizenz-Keys&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>110</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>LizenzName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>AblaufDatum</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>LizenzSchlüssel</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Bemerkung</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>Windows Lizenz</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>28.04.2023</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>dfg-dfg-arw-234-dgdf-h54whtr</string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>nur ein Placeholder</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>Backup</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>01.01.1970</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>456546-dsgdsf-13ref-34tge</string>
       </property>
      </item>
      <item row="1" column="3">
       <property name="text">
        <string>xXx</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="0" rowspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="newEntryButton">
        <property name="text">
         <string>Neuen Eintrag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Speichern</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>Eintrag Löschen</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMen">
    <property name="title">
     <string>Menü</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMen"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
